addons/payment_asiapay/models/payment_provider.py

```diff
@@ -17,6 +17,14 @@ def _domain_asiapay_currency_id(self):
     code = fields.Selection(
         selection_add=[('asiapay', "AsiaPay")], ondelete={'asiapay': 'set default'}
     )
+    asiapay_brand = fields.Selection(
```

Comment:
Stored fields can not be added in stable versions.

Reply:
In this case what should I do? I cant make it compute.

Reply:
IMHO the best course of action is to update the PR to target master (16.2) rather than 16.0.

New users will have the feature as soon as the version is released on Odoo Online, and upgrading users will get it a few weeks later. Odoo.sh and on-premise users will get the feature when Odoo 17 is released.

There are other ways to alter stable versions, but they are reserved for bugfixes. We usually avoid them for feature additions.

Reply:
That make sense, allow me to discuss with FJA and come back to you tomorrow.

Reply:
I created a new PR targeting master. Thanks!